From b5937f85fb78a141575a2c60d655373b4f1be099 Mon Sep 17 00:00:00 2001
From: Todd Zullinger <tmz@pobox.com>
Date: Wed, 28 Jan 2009 01:10:06 -0500
Subject: [PATCH 1/3] cuetag.sh: Fix metaflac options for flac >= 1.1.3

The --remove-vc-all and --import-vc options were removed in flac-1.1.3
(27-Nov-2006).  They were deprecated since flac-1.1.1 (01-Oct-2004).
Use the options which replaced them, -remove-all-tags and
--import-tags-from, respectively.
---
 src/tools/cuetag.sh |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

Index: cuetools-1.3.1/extras/cuetag.sh
===================================================================
--- cuetools-1.3.1.orig/extras/cuetag.sh
+++ cuetools-1.3.1/extras/cuetag.sh
@@ -18,7 +18,7 @@ vorbis()
 {
 	# FLAC tagging
 	# --remove-vc-all overwrites existing comments
-	METAFLAC="metaflac --remove-vc-all --import-vc-from=-"
+	METAFLAC="metaflac --remove-all-tags --import-tags-from=-"
 
 	# Ogg Vorbis tagging
 	# -w overwrites existing comments
